lib/thread.js

```diff
@@ -31,12 +31,13 @@ function createReporterOptions(string) {
 }
 
 function createReporterConfigFile(path) {
-  const reporterEnabled = ['@openx/cypress-parallel/json-stream.reporter.js'];
+  console.log('Creating reporter config file', __dirname);
```

Comment:
do we need this console log? it's more difference with upstream that is not necessary